dev-requirements.txt

```diff
@@ -1,5 +1,6 @@
 coverage
 invoke
+mock>=1.3.0
```

Comment:
This may be cavalier, but I don't see the need for dev requirements.  The mock requirement is to avoid a bug in mock that affects python 3.4.  I should add a comment describing this.